jsonschema-core.xml

```diff
@@ -219,6 +223,12 @@
                             <t hangText="string:">A string of Unicode code points, from the JSON "string" value</t>
                         </list>
                     </t>
+                    <t>
+                        Binary data MAY be treated as an instance, however no data type in the data model
+                        is suitable.  Therefore, only schemas such as the empty schema that do not
```

Comment:
The added paragraph above this one says that it's possible there's some application-defined mapping from binary data to the JSON data model. The wording of this paragraph seems to waffle between "MAY be treated as an instance" and "can't do anything unless it's an empty schema". Maybe add some language that suggests that a schema can be applied if that data model is applied. For example (changes in bold), "Binary data MAY be treated as an instance, however no data type in the data model is **directly** suitable. Therefore, only schemas such as the empty schema that do not constrain the type can be considered to pass **outright**."

I'm sure there's some better language, but this paragraph feels like it's fighting with itself if there's no acknowledgement that there exists that "application-defined mapping" in the previous paragraph.